go-guru.el

```diff
@@ -117,7 +117,7 @@
   (define-key m ">" #'go-guru-callees)
   (define-key m "x" #'go-guru-expand-region)) ;; x for expand
 
-(define-key go-mode-map (kbd "C-c C-o") #'go-guru-map)
+(define-key go-mode-map (kbd "C-c C-o") 'go-guru-map)
```

Comment:
"Function might not be defined". Indeed `go-guru-map` isn't a real function that could be called, but a keymap.